kore-rpc-types/src/Kore/Network/JsonRpc.hs

```diff
@@ -0,0 +1,175 @@
+{-# LANGUAGE TypeApplications #-}
+
+{- |
+Module      : Kore.Network.JSONRPC
+Description : JSON RPC server for kore-rpc
+Copyright   : (c) Runtime Verification, 2022
+License     : BSD-3-Clause
+-}
+module Kore.Network.JsonRpc (jsonRpcServer, JsonRpcHandler (..)) where
```

Comment:
Maybe we could use a more intuitive name?
And we could re-export types that everyone will need from `Data.Network.JsonRpc`...
```suggestion
module Kore.JsonRpc.Server (
    ErrorObj (..),
    Request (..),
    Respond,
    jsonRpcServer,
    JsonRpcHandler (..),
) where
```